Classes/common/CDTIncrementalStore.m

```diff
@@ -27,7 +27,7 @@
 #import "CDTISGraphviz.h"
 
 #ifdef HAS_NSBatchUpdateRequest
-/**
```

Comment:
Oh, that makes sense. I don't see a better way right now, excluding certain methods doesn't seem possible. I'm fine with merging this code in.